tests/integration/test_config.py

```diff
@@ -339,3 +378,381 @@ async def test_host_group_config(cluster: Cluster) -> None:
     param: ParameterHG = config_1["more"]["strange"]  # type: ignore
     # bit of ADCM logic: sync parameter shipped from object's config
     assert param.value == strange_val_1
+
+
+async def test_config_two_sessions(
+    adcm: ADCMContainer, httpx_client: AsyncClient, cluster: Cluster, simple_hostprovider_bundle: Bundle
+) -> None:
+    kwargs = {"verify": False, "timeout": 10, "retry_interval": 1, "retry_attempts": 1}
+    credentials = Credentials(username="admin", password="admin")  # noqa: S106
+
+    async with ADCMSession(url=adcm.url, credentials=credentials, **kwargs) as second_client:
+        provider = await second_client.hostproviders.create(bundle=simple_hostprovider_bundle, name="Test HP")
+        host = await second_client.hosts.create(hostprovider=provider, name="Test-host", cluster=cluster)
+
+        # get two copies of the same object with different requesters
```

Comment:
clients, not requesters, in this context

Reply:
fixed